src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb

```diff
@@ -412,6 +412,37 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 statements.Add(New BoundReturnStatement(methodBlock.EndBlockStatement, Nothing, Nothing, Nothing))
             End If
 
+            ' Inject implicit base constructor call, if appropriate
+            If Not methodSymbol.IsImplicitlyDeclared AndAlso methodSymbol.MethodKind = MethodKind.Constructor Then
+                Dim referencedConstructor As MethodSymbol = Nothing
+                Dim injectDefaultConstructorCall As Boolean = False
+
+                MethodCompiler.GetExplicitlyOrImplicitlyReferencedConstructor(methodSymbol, statements(0), Me,
```

Comment:
>statements(0) [](http://example.com/codeflow?start=92&length=13)

Is there always at least one statement at this point?

Reply:
> Is there always at least one statement at this point?

Yes, there is at least a `BoundReturnStatement` added in both branches of the previous `If`.